mypy_django_plugin/transformers/managers.py

```diff
@@ -56,11 +82,24 @@ def get_funcdef_type(definition: Union[FuncBase, Decorator, None]) -> Optional[P
         return None
 
     assert isinstance(method_type, CallableType)
+
+    variables = method_type.variables
+    ret_type = method_type.ret_type
+
+    # For methods on the manager that return a queryset we need to override the
+    # return type to be the actual queryset class, not the base QuerySet that's
+    # used by the typing stubs.
+    if method_name in MANAGER_METHODS_RETURNING_QUERYSET:
```

Comment:
Maybe `MANAGER_METHODS_RETURNING_QUERYSET` can be a `frozenset`?

Reply:
Fixed this now, though I don't think it matters much performance wise. Can't be bothered to run any tests though 😇